ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -43,12 +61,57 @@ const PersonalSignInfo: React.FC = () => {
     <>
       {isSIWE && useTransactionSimulations && (
         <ConfirmInfoSection>
-          <ConfirmInfoRow
-            label={t('simulationDetailsTitle')}
-            tooltip={t('simulationDetailsTitleTooltip')}
+          <Box
+            data-testid="simulation-details-layout"
+            className="simulation-details-layout"
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            borderRadius={BorderRadius.LG}
+            borderColor={BorderColor.transparent}
+            padding={2}
+            gap={3}
           >
-            <ConfirmInfoRowText text={t('siweSignatureSimulationDetailInfo')} />
-          </ConfirmInfoRow>
+            <Box
+              display={Display.Flex}
+              flexDirection={FlexDirection.Row}
+              alignItems={AlignItems.center}
+              justifyContent={JustifyContent.spaceBetween}
+            >
+              <Box
```

Comment:
Minor, since we have three nested boxes, would the readability benefit from some local components?